kanopi/kanopi-pack
# PostCSS and Asset Options

## Description
> As a package maintainer, I need to enable CSS only compilation for Kanopi Pack
> As a package maintainer, I need to enable access for static assets through CSS only, as it's currently only open to SASS compilation

## Steps to Validate
1. Install the PR version of KP for this feature branch `npm i https://github.com/kanopi/kanopi-pack.git#feature/postcss-options` for a project which uses SCSS.
2. Set the option in KP Config to `styles.useSass: false`. Run a build and confirm it breaks the build.
3. Install it into a project with only CSS which uses some of the supplied PostCSS Components, like a project using HiQ.
4. Set the option in KP Config to `styles.useSass: false`. Run a build and confirm the project builds and styles display as expected.
5. Add an image of svg into a folder, like `assets/src/static/images/icon.svg`
6. Add this image through CSS using the `url('@/static/images/icon.svg')` syntax and verify the image loads through both Production and Development mode.
